wipp-backend-data/src/main/java/gov/nist/itl/ssd/wipp/backend/data/csvCollection/CsvCollectionDataHandler.java

```diff
@@ -72,6 +72,14 @@ public String exportDataAsParam(String value) {
         }
         // else return the path of the csv collection
         else {
+            if(csvCollectionRepository.findById(csvCollectionId).isPresent()) {
```

Comment:
`csvCollectionRepository.findById(csvCollectionId)` could be called only once before the if and stored in a variable (to avoid calling it twice)